src/main/java/org/jenkinsci/plugins/pipeline/utility/steps/zip/UnZipStepExecution.java

```diff
@@ -175,8 +184,18 @@ public UnZipFileCallable(TaskListener listener, FilePath destination, String glo
                     }
                 }
                 if (read) {
+                    if (quiet) {
+                        logger.print("Read: ");
```

Comment:
This would be good to have be printed even if `quiet` is false

Reply:
That can certainly be done. I'll have a new commit with that momentarily.

Reply:
With that change the tests I added will always work, even if run with quiet = false, since they were just looking for that. Is there an assertLogNotContains or anything like that I could use to make the test check that "Extracting: hello.txt ->" is not in the log?

Reply:
Yes https://github.com/jenkinsci/jenkins-test-harness/blob/master/src/main/java/org/jvnet/hudson/test/JenkinsRule.java#L1255

Reply:
Thanks! Tests have been updated with that checking.